packages/fxa-auth-server/lib/server.js

```diff
@@ -173,8 +173,28 @@ async function create(log, error, config, routes, db, statsd, glean) {
   const server = new Hapi.Server(serverOptions);
   server.validator(require('joi'));
 
-  server.ext('onRequest', (request, h) => {
+  const SKIP_CUSTOMS_FOR = [
+    '__lbheartbeat__',
+    'config',
+    '__heartbeat__',
+    '__version__',
+    '',
+  ];
+
+  server.ext('onRequest', async (request, h) => {
     log.begin('server.onRequest', request);
+
+    // Should fallback to the default rate-limit rule, or if no
+    // default rule is configured, then no check is done.
+    if (customs && !SKIP_CUSTOMS_FOR.includes(request.path)) {
```

Comment:
On the fence about this, but maybe we should also add a separate config to disable this entirely?

Reply:
@vbudhram I've added this and some other config so we can adjust how this works as necessary.